### ISSUE TITLE
Allow user to choose json sources for data

### ISSUE THREAD

**Issue by ryuran** [kss-node/kss-node#402]
Sometimes I would display a component (twig template) with different data.
But currently the only way to link data to template is naming json file and twig file with the same name.

**Comment by JohnAlbin** [comment]
Yeah, I've thought something like this would be nice. A related feature would be a way to add variable length content, especially useful for testing headlines that wrap.
